VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioComponentDetails.cpp

```diff
@@ -459,8 +573,25 @@ void AddScalarWidget(const TArray<TSharedRef<IDetailTreeNode>> DetailTreeNodes,
 	}
 }
 
+bool DifferentValuesSelected(URuleAttribute* Attribute, FString AttributeKey, TArray<UVitruvioComponent*> VitruvioComponents)
```

Comment:
Parameters could be passed by const reference.

Reply:
Also consider renaming the function to something more descriptive (the selected postfix is only "implied" by the callsite).

Maybe something like `AreValuesDifferent` (or something like that, up to you).